site/source/_posts/2018/01/17-Announcing-Dojo-2-beta-5/Announcing-Dojo-2-beta-5.md

```diff
@@ -0,0 +1,109 @@
+
+## Just getting started
+
+If you want to get started with Dojo 2, the best way is to take a look at our [tutorials](/tutorials/) or even take a look at Dojo 2 in action with our [example applications](https://dojo.github.io/examples/) (source on [GitHub](https://github.com/dojo/examples)).  Good starting tutorials are:
+
+* [Dojo 2 local installation](/tutorials/000_local_installation/)
+* [Your first Dojo 2 application](/tutorials/001_static_content/)
+* [Creating widgets](/tutorials/003_creating_widgets/)
+
+## New in beta 5
+
+Since the beta 4 release, we have made a number of improvements. The biggest changes and additions are mentioned here, with a complete list of changes
+
+### Initial re-architecture of the build system
+
+Prior to beta5, the structure of the build commands was focused around the tooling and not around the actual creation of applications. So we completed a re-architecture and restructuring of the commands to align more to their use. Specifically, `@dojo/cli-build-webpack` is deprecated and replaced with:
+
+* [`@dojo/cli-build-app`](https://github.com/dojo/cli-build-app/) A command that builds an application
+* [`@dojo/cli-build-widget`](https://github.com/dojo/cli-build-widget/) A command that build a widget, including support for web components
+* Other commands may be planned for the future
+
+This refactoring is mostly complete for Beta 5, with a few remaining items to complete for release candidate 1.
+
+Usage of the build system is reflected in updates to the [Dojo 2 tutorials](https://dojo.io/tutorials/).
+
+### Improvements to using Custom Elements/Web Components
+
+TODO: Explain
+
+### widget-core performance and memory improvements
+
+TODO: Explain, provide metrics?
+
+### New widgets
+
+Two new widgets were added in Beta 5:
+
+* [Progress](https://github.com/dojo/widgets/issues/385) (`@dojo/widgets/progress/Progress`)
+* [Toolbar](https://github.com/dojo/widgets/issues/386) (`@dojo/widgets/toolbar/Toolbar`)
+
+TODO Add screenshots
+
+These widgets may be viewed in the [widget showcase](https://dojo.github.io/examples/widget-showcase/).  TODO: blocked by https://github.com/dojo/examples/issues/278
+
+### Stores
+
+A number of refinements were made since the initial `@dojo/stores` release in Beta 4:
+
+* [Add a basic `onChange` mechanism](https://github.com/dojo/stores/commit/b6165e6fd670874726e67ab166cb888967785d19)
+* Improvements to providing type definitions for state
+
+See the [@dojo/stores readme](https://github.com/dojo/stores) for more information about this package.
+
+### Updated examples
+
+Our collection of [Dojo 2 examples](http://github.com/dojo/examples/) were updated for beta 5.
+
+In particular, the [kitchen sink version of ToDoMVC](https://github.com/dojo/examples/blob/master/todo-mvc-kitchensink) now uses `@dojo/stores`.
```

Comment:
We could potentially say our hnpwa example was accepted?

Reply:
have a link handy?

Reply:
to the running application or the hnpwa website? or both?

Reply:
So I looked at https://hnpwa.com/ but didn't see the Dojo app... I see it now at https://dojo-2-hnpwa-d668d.firebaseapp.com/